src/components/scrollBar/index.tsx

```diff
@@ -25,7 +25,7 @@ export interface ScrollBarProps extends FlatListProps<any> {
     /**
      * The element to use as a container, instead of a View
      */
-    containerView?: JSX.Element,
+    containerView?: object,
```

Comment:
Is that right? 
Shouldn't `containerView` be an element?

Reply:
No. Since we're not passing <TabletView/>, we're passing TabletView.

Reply:
Still it's not a object. 
I think there's a dedicate type for it.. 

Try searching for the right prop.

Reply:
I couldn't find it, the 'object' type is the only one that doesn't error...